### DISCUSSION TITLE
Navigation within a module

### DISCUSSION THREAD

**Discussion by tiggertag** [oqtane/oqtane.framework#2344] [Q&A] [answered]
I am just getting my feet wet with Oqtane, also, with Blazor. So far, on the modules, I created a new module with Module Creator, which shows a list of items on the page and and a way to add a new item. When I click on add new item button, I see the navigation happens with ActionLink , which opens up a closable popup window which loads a form to enter the item information. Sorry, this might be a dumb question, but, is there a way to navigate without the popup? What I mean is for example, when you go to Discussions on Github, you will see the list of discussions and  you click to create a new discussion, the form opens up without a closable popup. How can I achieve the same thing in the Oqtane.

**Comment by leigh-pointer** [comment, marked as answer]
https://github.com/oqtane/oqtane.framework/discussions/2281#discussioncomment-3140536 Think this will help
________________________________
From: tiggertag ***@***.***>
Sent: Sunday, August 7, 2022 5:59:39 PM
To: oqtane/oqtane.framework ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [oqtane/oqtane.framework] Navigation within a module (Discussion #2344)


I am just getting my feet wet with Oqtane, also, with Blazor. So far, on the modules, I created a new module with Module Creator, which shows a list of items on the page and and a way to add a new item. When I click on add new item button, I see the navigation happens with ActionLink , which opens up a closable popup window which loads a form to enter the item information. Sorry, this might be a dumb question, but, is there a way to navigate without the popup? What I mean is for example, when you go to Discussions on Github, you will see the list of discussions and you click to create a new discussion, the form opens up without a closable popup. How can I achieve the same thing in the Oqtane.
